docs/tools/visual-studio-code/sql-server-develop-use-vscode.md

```diff
@@ -71,7 +71,7 @@ Follow these steps to create a connection profile and connect to a SQL Server.
 
    | Connection property | Description |
    |---|---|
-   | **Server name or ADO connection string** | Specify the SQL Server instance name. Use *localhost* to connect to a SQL Server instance on your local machine. To connect to a remote SQL Server, enter the name of the target SQL Server, or its IP address. To connect to a SQL Server container, specify the IP address of the container's host machine. If you need to specify a port, use a comma to separate it from the name. For example, for a server listening on port 1401, enter `<servername or IP>,1401`.<br/><br/>As an alternative, you can enter the ADO connection string for your database here. |
+   | **Server name or ADO connection string** | Specify the SQL Server instance name. Use *localhost* to connect to a SQL Server instance on your local machine. To connect to a remote SQL Server, enter the name of the target SQL Server, or its IP address. To connect to a SQL Server container, specify the IP address of the container's host machine. If you need to specify a port, use a comma to separate it from the name. For example, for a server listening on port 1401, enter `<servername or IP>,1401`.<br/><br/>When port is not specified, its default value `1433`, which is also the default port of SQL Server, will be used implicitly. Hence, specifying the port explicitly is unnecessary if you haven't change the configuration of SQL Server.<br/><br/>As an alternative, you can enter the ADO connection string for your database here. |
```

Comment:
@kmivan - thanks for the suggestion. I'm thinking about updating as follows to simplify the language a little. I also want to be clear that it doesn't matter who may have changed the configuration.

```suggestion
   | **Server name or ADO connection string** | Specify the SQL Server instance name. Use *localhost* to connect to a SQL Server instance on your local machine. To connect to a remote SQL Server, enter the name of the target SQL Server, or its IP address. To connect to a SQL Server container, specify the IP address of the container's host machine. If you need to specify a port, use a comma to separate it from the name. For example, for a server listening on port 1401, enter `<servername or IP>,1401`.<br/><br/>By default, the connection string uses port 1433. A default instance of SQL Server uses 1433 unless modified. If your instance is listening on 1433, you do not need to specify the port.<br/><br/>As an alternative, you can enter the ADO connection string for your database here. |
```